kairon/shared/channels/whatsapp/bsp/gupshup.py

```
@@ -0,0 +1,483 @@
+
+            payload = data.copy()
+            payload['buttons'] = json.dumps(payload.get('buttons'))
+            headers = {
+                header: partner_app_token,
+                "Content-Type": "application/x-www-form-urlencoded",
+                "accept": "application/json"
+            }
+
+
+            url = f"{partner_base_url}/partner/app/{app_id}/templates"
+            resp = requests.post(url, headers=headers, data=payload)
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)